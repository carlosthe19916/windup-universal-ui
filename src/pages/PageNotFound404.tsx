import * as React from "react";
import {
  PageSection,
  Bullseye,
  EmptyState,
  EmptyStateVariant,
  Title,
  EmptyStateIcon,
} from "@patternfly/react-core";
import { UnknownIcon } from "@patternfly/react-icons";

export const PageNotFound404: React.FC = ({ children }) => {
  return (
    <React.Fragment>
      <PageSection>
        <Bullseye>
          <EmptyState variant={EmptyStateVariant.full}>
            <EmptyStateIcon icon={UnknownIcon} />
            <Title headingLevel="h5" size="lg">
              Error 403 Not found! The resource you are trying to reach does not
              exists.
            </Title>
          </EmptyState>
          {children}
        </Bullseye>
      </PageSection>
    </React.Fragment>
  );
};
