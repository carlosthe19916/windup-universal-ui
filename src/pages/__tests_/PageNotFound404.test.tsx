import * as React from "react";
import { shallow } from "enzyme";
import { PageNotFound404 } from "../PageNotFound404";

it("Renders without crashing", () => {
  const action = jest.fn();
  const wrapper = shallow(<PageNotFound404 />);

  expect(wrapper).toMatchSnapshot();
});
