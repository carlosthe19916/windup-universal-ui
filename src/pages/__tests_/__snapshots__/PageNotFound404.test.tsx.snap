// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<Fragment>
  <PageSection>
    <Bullseye>
      <EmptyState
        variant="full"
      >
        <EmptyStateIcon
          icon={[Function]}
        />
        <Title
          headingLevel="h5"
          size="lg"
        >
          Error 403 Not found! The resource you are trying to reach does not exists.
        </Title>
      </EmptyState>
    </Bullseye>
  </PageSection>
</Fragment>
`;
